content/docs/buildpacks/language-family-buildpacks/nodejs.md

```diff
@@ -223,6 +223,29 @@ BP_LAUNCHPOINT=./src/launchpoint.js
 
 This will result in the following start command: `node src/launchpoint.js`
 
+## Specifying Project Directory
+
+To specify a subdirectory to be used as the root of the app, please use the
+`BP_NODE_PROJECT_PATH` environment variable at build time either directly or
+through a [`project.toml`](https://github.com/buildpacks/spec/blob/main/extensions/project-descriptor.md).
```

Comment:
Could we point to public docs instead? https://buildpacks.io/docs/app-developer-guide/using-project-descriptor/

Reply:
done